metrics/testrecorder/testrecorder.go

```diff
@@ -24,8 +24,12 @@ func (tr *TestMetricsRecorder) Handler() http.Handler {
 	return nil
 }
 
-func (tr *TestMetricsRecorder) ObserveTask(task *tasks.Task) error {
-	tr.tasks[task.UUID] = append(tr.tasks[task.UUID], task.Status)
+func mustString(s string, _ error) string {
```

Comment:
maybe export this in the tasks package? not that the copy-pasting is significant, but it seems like an IPLD helper so it could just go in there too. And it's already been copied a few times.